PubHub/PubHub.Common/Services/ServiceResult.cs

```diff
@@ -10,7 +10,6 @@ internal ServiceResult(HttpStatusCode statusCode, string errorDescriptor)
             ErrorDescriptor = errorDescriptor;
         }
 
-        public bool IsSuccess => ((int)StatusCode >= 200) && ((int)StatusCode <= 299);
```

Comment:
Why did we remove IsSuccess?

Reply:
It wasn't used anywhere and it gave a warning when setting an attribute over the `T Instance` in the `ServiceReault{TResult}`

Reply:
But "wasn't used anywhere" can only be said for your project (:
You don't know whether other projects use that particullar line of code, which means a PR could be broken if it uses that line (_mine does_).

We never just remove shared code. It's bad practise as several systems could depend on it.
And it's easier to check `IsSuccess` than to check the status code. This is also why `HTTPResponse.IsSuccess` exists (:

It's fine, though htat your removed the attribute as that clearly didn't work, however, I don't feel like because an attribute couldn't make use of `IsSuccess` that a removal of the property was warrented 🤔

Reply:
We added the `IsSuccess` because of that but, it makes sense to use it regardless so I agree with your perspective and will bring it back into the the `ServiceResult.cs`. 😊